package.json

```diff
@@ -30,14 +31,16 @@
     "connect-history-api-fallback": "^1.2.0",
     "del": "^2.2.1",
     "es6-module-loader": "^0.17.11",
+    "eslint": "^2.13.1",
+    "eslint-config-sedona": "sedona-solutions/eslint-config-sedona",
```

Comment:
The eslint-config-sedona should be publish to the public npm repository. Then, use the right version.